models/restaurants.go

```diff
@@ -3,19 +3,22 @@ package models
 import (
 	"database/sql"
 	"fmt"
+	"time"
 )
 
 // Restaurant data struct
 type Restaurant struct {
-	ID      int            `json:"id"`
-	Name    string         `json:"name"`
-	Address string         `json:"address"`
-	Number  sql.NullString `json:"number"`
+	ID        int            `json:"id"`
+	Name      string         `json:"name"`
+	Address   string         `json:"address"`
+	Number    sql.NullString `json:"number"`
+	CreatedAt time.Time      `json:"created_at"`
+	UpdatedAt time.Time      `json:"updated_at"`
```

Comment:
It seems like the order of these attributes matter

Reply:
Why do we need the time format? Type string is working,  and I don't know what is going to be when you send it to the front end. I am guessing your database is not correct or different than mine.

Reply:
![Screen Shot 2020-03-26 at 9 38 09 PM](https://user-images.githubusercontent.com/8388269/77712444-29861d00-6faa-11ea-91c5-4f0fc7ec364f.png)
<img width="385" alt="Screen Shot 2020-03-26 at 9 38 16 PM" src="https://user-images.githubusercontent.com/8388269/77712446-2be87700-6faa-11ea-9d17-7dc43ba5ceba.png">

Reply:
Time is time. `time` format gives us control over how we want to format it. 

https://golang.org/pkg/time/